vembrane/modules/annotate.py

```diff
@@ -0,0 +1,129 @@
+import contextlib
+import csv
+import sys
+import yaml
+import pandas as pd
```

Comment:
make sure to include newly added dependencies to the pyproject.toml, such as pandas!

Reply:
and removed